app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/provision/options_helper.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers::IbmCloud::VPC::CloudManager::Provision::OptionsHelper
+  # The ID for this EMS.
+  # @return [Integer]
+  def cloud_instance_id
+    source.ext_management_system.uid_ems
```

Comment:
Are you trying to return the `:id` or the `:uid_ems` here?  The comment indicates you want the `:id` but this is returning the `:uid_ems` string.

PowerVS sets `:uid_ems` to the pcloud_guid but I don't believe VPC sets this to anything.

Reply:
Actually is the `#cloud_instance_id` method used anywhere here?

Reply:
Dunno, that is another copy paste from Power.

Reply:
Okay this isn't needed for VPC and should be removed then